packages/terra-functional-testing/src/commands/hide-input-caret.js

```diff
@@ -0,0 +1,23 @@
+const Logger = require('../logger/logger');
+
+const logger = new Logger({ prefix: 'wdio-terra-service' });
+
+/**
+ * Hides the blinking input caret that appears within editable text areas to prevent inconsistent test failures.
+ *
+ * @param {string} selector The selector for the element to hide the caret of
+ */
+const hideInputCaret = (selector) => {
+  try {
+    global.browser.execute(`document.querySelector("${selector.replace(/"/g, '\\"')}").style.caretColor = "transparent";`);
+  } catch (error) {
+    const element = global.browser.$(selector);
+    if (!element.isExisting()) {
+      logger.error(`No element could be found with the selector '${selector}'.`);
+    } else {
```

Comment:
Should we check if the element exists before trying to update the style so a try-catch isn't needed?

Reply:
Yes, I think that makes more logical sense. If the selector isn't there, we're just log the message and no need to throw an error.